modal-go/app_test.go

```diff
@@ -0,0 +1,67 @@
+package modal
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestParseGPUConfig(t *testing.T) {
```

Comment:
As I understand, putting `*_test.go` files among other package files is the standard way to unit test private functions in Go, is that right @luiscape @thomasjpfan ?

Reply:
Yes, but you'll need to update the following to run the test in CI:

https://github.com/modal-labs/libmodal/blob/261c8a327283f57cf44507afbc29f6c35a7395e6/.github/workflows/ci.yaml#L100

```bash
go test -v -count=1 -parallel=10 ./...
```